packages/react-core/src/next/components/Modal/ModalBoxBody.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/ModalBox/modal-box';
+
+export interface ModalBoxBodyProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content rendered inside the modal box body. */
+  children?: React.ReactNode;
+  /** Additional classes added to the modal box body. */
+  className?: string;
+  /** Accessible label applied to the modal box body. This should be used to communicate
+   * important information about the modal box body div element if needed, such as that it
+   * is scrollable.
+   */
```

Comment:
Rather than implying that the aria-label should mention "This container is scrollable", we should convey that it should be passed when the container is scrollable

```suggestion
  /** Accessible label applied to the modal box body. This should be used to communicate
   * important information about the modal box body div element if needed, such as when it is scrollable.
   */
```